pkgs/applications/graphics/photoflow/default.nix

```diff
@@ -1,42 +1,79 @@
-{ stdenv, fetchFromGitHub, gettext, glib, libxml2, pkgconfig, swig, automake, gobject-introspection, cmake, ninja, libtiff, libjpeg, fftw, exiv2, lensfun, gtkmm2, libraw, lcms2, libexif, vips, expat, pcre, pugixml }:
+{ automake
+, cmake
+, exiv2
+, expat
+, fetchFromGitHub
+, fftw
+, fftwFloat
+, gettext
+, glib
+, gobject-introspection
+, gtkmm2
+, lcms2
+, lensfun
+, libexif
+, libiptcdata
+, libjpeg
+, libraw
+, libtiff
+, libxml2
+, ninja
+, openexr
+, pcre
+, pkgconfig
+, pugixml
+, stdenv
+, swig
+, vips
+}:
 
-stdenv.mkDerivation {
-  name = "photoflow-unstable-2018-08-28";
+stdenv.mkDerivation rec {
+  pname = "photoflow";
+  version = "2020-08-28";
 
   src = fetchFromGitHub {
     owner = "aferrero2707";
-    repo = "PhotoFlow";
-    rev = "df03f2538ddd232e693c307db4ab63eb5bdfea38";
-    sha256 = "08ybhv08h24y4li8wb4m89xgrz1szlwpksf6vjharp8cznn4y4x9";
+    repo = pname;
+    rev = "8472024fb91175791e0eb23c434c5b58ecd250eb";
+    sha256 = "1bq4733hbh15nwpixpyhqfn3bwkg38amdj2xc0my0pii8l9ln793";
   };
 
+  patches = [ ./CMakeLists.patch ];
+
   nativeBuildInputs = [
+    automake
+    cmake
     gettext
     glib
+    gobject-introspection
     libxml2
+    ninja
     pkgconfig
     swig
-    automake
-    gobject-introspection
-    cmake
-    ninja
   ];
 
   buildInputs = [
-    libtiff
-    libjpeg
-    fftw
     exiv2
-    lensfun
+    expat
+    fftw
+    fftwFloat
     gtkmm2  # Could be build with gtk3 but proper UI theme is missing and therefore not very usable with gtk3
             # See: https://discuss.pixls.us/t/help-needed-for-gtk3-theme/5803
-    libraw
     lcms2
+    lensfun
     libexif
-    vips
-    expat
+    libiptcdata
+    libjpeg
+    libraw
+    libtiff
+    openexr
     pcre
     pugixml
+    vips
+  ];
+
+  configureFlags = [
+    "--${if stdenv.hostPlatform.sse3Support   then "enable" else "disable"}-sse3"
```

Comment:
```suggestion
    "--${if stdenv.hostPlatform.sse3Support then "enable" else "disable"}-sse3"
```

Reply:
Will update in a bit, still waiting to see if this fixes the aarch64 build, otherwise I'll just remove it and mark aarch64 as a `badPlatform`

Reply:
Didn't fix it, removed